util/mk1mf.pl

```diff
@@ -304,6 +304,7 @@
 $cflags.=" -DOPENSSL_NO_ASYNC" if $no_async;
 $cflags.=" -DOPENSSL_NO_AUTOALGINIT" if $no_autoalginit;
 $cflags.=" -DOPENSSL_NO_AUTOERRINIT" if $no_autoerrinit;
+$cflags.=" -DOPENSSL_NO_NEXTPROTONEG" if $no_nextprotoneg;
```

Comment:
@levitte probably knows this better, but I think it also requires a change in the valid_options